app/services/teacher_training_api/import_course.rb

```diff
@@ -39,19 +37,27 @@ def call
         study_mode: course_attributes[:study_mode],
         accredited_body_code: accredited_body_code,
         recruitment_cycle_year: Settings.current_recruitment_cycle_year,
-      )
+      }.merge(course_age_attributes))
+
+      course
     end
 
   private
 
     attr_reader :course_data, :provider_data, :course_attributes
 
-    def further_education_level_course?
-      course_attributes[:level] == "further_education"
+    def course_age_attributes
+      return {} if primary_level_course? && course_attributes[:age_maximum] > AgeRange::UPPER_BOUND_PRIMARY_AGE
```

Comment:
Do we need to guard against `course_attributes[:age_maximum] = nil`?
```suggestion
      return {} if primary_level_course? && course_attributes[:age_maximum].to_i > AgeRange::UPPER_BOUND_PRIMARY_AGE
```

Reply:
I don't think we every had a course without an age range, but happy to put in the guard just in case.